src/system_structure.jl

```diff
@@ -434,7 +439,7 @@ A rigid wing body that can have multiple groups of points attached to it.
 - `group_idxs::Vector{Int16}`: Indices of groups attached to this wing
 - `transform_idx::Int16`: Transform used for initial positioning and orientation
 - `R_b_c::Matrix{SimFloat}`: Rotation matrix from body frame to CAD frame
-- `angular_vel::KVec3`: Angular velocity of the wing in world frame
+- `ω_b::KVec3`: Angular velocity of the wing in world frame
```

Comment:
There is no such thing as a world frame. Please, use only names of reference frames that are defined in our documentation.